testrocket.gemspec

```diff
@@ -1,21 +1,21 @@
 # -*- encoding: utf-8 -*-
-$:.push File.expand_path("../lib", __FILE__)
-require "testrocket"
+$:.push File.expand_path('../lib', __FILE__)
+require 'testrocket'
 
 Gem::Specification.new do |s|
-  s.name        = "testrocket"
+  s.name        = 'testrocket'
   s.version     = TestRocket::VERSION
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Peter Cooper","Christoph Grabo"]
-  s.email       = ["git@peterc.org","chris@dinarrr.com"]
-  s.homepage    = "http://github.com/peterc/testrocket"
+  s.authors     = %w[Peter Cooper Christoph Grabo]
```

Comment:
This needs to go back to the original, as it creates an array of four elements due to the spaces.

Reply:
Oh my! Sure thing. Gonna change it back.